### PR TITLE
feat(python and typescript): RDS MySQL, Oracle, and Aurora samples

### PR DESCRIPTION
**Pull request by DevoKun**
## Python

* Moved the original `rds` subdirectory to [`rds/mysql`](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-839ce22be20ea5ee94f5b336a323c5c2aa4fa14a079b6bab1e35ffaa8a3e8a9e), added a [readme](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-db7445cf5800f6463096c20d26eb9e01c0e38c7f20fd703362eb489103e32c79), and expanded the construct to take parameters passed to the class init.
* Added an [RDS Aurora Construct](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-ceabaf456adb7ddf011068368a7fe47673c87e568691af4bb23c80d3d7e4f54b)
* Added an [RDS Oracle Construct](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-8e1b2dd5189366cc3767640c212632a2630616e696d91c8298a4d5314baf426e)
* Created a [parent readme for all RDS](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-b2ffba206a086573b1b7162c71e882560e52ae2d494ea7faa9d85df9a9469bb1).

## Typescript

* Added an [RDS Aurora Construct](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-93da52920ea84c13e34905ecdf8fe20ac71f1d8fd48fcd27da514edac7674d6d)
* Added an [RDS MySQL Construct](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-d7fa3f7f016b8020a55ea61522bce38fbefa1546ae2139a986237f64fc768c9b)
* Added an [RDS Oracle Construct](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-7a48dee129e7bf2963f60742067d0100aa55405e30e9dc15a00110789f2bfa8e)
* Created a [parent readme for all RDS](https://github.com/aws-samples/aws-cdk-examples/pull/654/files#diff-64ec6504a2ddd7e92e22ddacc0d7746ee6b320b5de47f2125c6e2fab64ab4734).

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: python/rds/oracle/oracle.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    CfnOutput(self, "OracleUsername",   
+      export_name="OracleUsername",   
+      value      =oracle_username)
+    CfnOutput(self, "OracleDbName",   
+      export_name="OracleDbName",   
+      value      =db_name)
+
+
+
+class LavaPlainsOfMustafar(Stack):
```
Comment (ryparker) [inline python/rds/oracle/oracle.py:217]: Love the name.

File: typescript/rds/aurora/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.0.0",
+    "constructs": "^10.0.0",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "source-map-support": "^0.5.16"
+  }
```
Comment (ryparker) [inline typescript/rds/aurora/package.json:25]:
```suggestion
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "*",
    "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "^2.0.0",
    "constructs": "^10.0.0",
    "source-map-support": "^0.5.16"
  }
```

These 3 dependencies can be moved to "devDependencies". `jest` and `ts-jest` are for testing and `ts-node` could be something the developer has installed at the global scope (similar to `typescript`).

File: typescript/rds/mysql/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.0.0",
+    "constructs": "^10.0.0",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "source-map-support": "^0.5.16"
+  }
```
Comment (ryparker) [inline typescript/rds/mysql/package.json:25]:
```suggestion
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "*",
    "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "^2.0.0",
    "constructs": "^10.0.0",
    "source-map-support": "^0.5.16"
  }
```

Same as other package.json

File: typescript/rds/oracle/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.0.0",
+    "constructs": "^10.0.0",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "source-map-support": "^0.5.16"
+  }
```
Comment (ryparker) [inline typescript/rds/oracle/package.json:25]:
```suggestion
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "*",
     "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "^2.0.0",
    "constructs": "^10.0.0",
    "source-map-support": "^0.5.16"
  }
```

Same as other package.json

**Review by ryparker — CHANGES_REQUESTED** [review]
This looks great. Thank you for putting in the effort and providing great documentation. I think CDK RDS users are going to love the examples you've provided.

Let me know if you have any questions about the about the ⛏️ nitpick I suggested in the `package.json`s. Then as long as the build succeeds we should be good to merge.

**Comment by DevoKun** [comment]
> This looks great. Thank you for putting in the effort and providing great documentation. I think CDK RDS users are going to love the examples you've provided.
> 
> Let me know if you have any questions about the about the ⛏️ nitpick I suggested in the `package.json`s. Then as long as the build succeeds we should be good to merge.

Thanks, Ryan.

I merged in your recommendations.

**Review by ryparker — DISMISSED** [review]

**Review by ryparker — APPROVED** [review]

**Review by ryparker — APPROVED** [review]

**Pull request merged**